### PR TITLE
Add 2 new mods, 1 person, and 3 langs

### PR DESCRIPTION
**Pull request by mugman174**

### CODE REVIEW

**Review by Ruthenic — CHANGES_REQUESTED** [review]

File: data.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
       "Strencher",
       "Swishilicous",
       "Tasky",
+      "The Commie Axolotl",
```
[the remark this answers is not in the document]
Comment (mugman174) [reply data.json:43]: [Velocity](https://velocity-discord.netlify.app/)

File: data.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
       "Kyza",
       "Lighty",
       "Molten",
-      "Mugman",
+      "mugman",
```
[the remark this answers is not in the document]
Comment (mugman174) [reply data.json]: fixed I think

File: data.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ruthenic) [reply data.json:43]: fair

**Review by Ruthenic — APPROVED** [review]
yeah looks fine to me

**Pull request merged**